src/cloud/components/onboarding/EngagementLink.tsx

```diff
@@ -0,0 +1,41 @@
+// Libraries
+import {FC, useEffect} from 'react'
+import {useLocation} from 'react-router-dom'
+import {useSelector} from 'react-redux'
+
+// Utils
+import {getOrg} from 'src/organizations/selectors'
+
+// Types
+import {AppState} from 'src/types'
+
+const EngagementLink: FC = () => {
+  const pathname = useLocation().pathname
+  const userpilot = window['userpilot']
```

Comment:
darn typescript being rude. if you find yourself making more references to this object like this, it might be a good idea to extend the window object in the globals.d.ts file, but no need to stress on a one off